src/ios/Accessibility.ts

```diff
@@ -89,6 +89,11 @@ export class Accessibility extends NativeAccessibility {
             }
         }
     }
+
+    private compareRawAnnouncements(postedAnnouncement: string, payloadAnnouncement: string): boolean {
+        const trimSpacesCommas = new RegExp(/[\s,]/);
```

Comment:
Is it required to replace only first occurrence?

```typescript
'test, test, tets, '.replace(/[\s,]/);
> 'test test, tets, '
```
or need to replace all spaces and commas?

```typescript
'test, test, tets, '.replace(/[\s,]/g);
> 'testtesttets'
```

Reply:
As long string is splitting in several with commas with spaces, better to remove them all for comparison

Example: 
```
const postedAnnouncement = "New message from Evgenii Utkin, please write me"
const payloadAnnouncement = "New message from ,  Evgenii Utkin, ,  please write me"
```

Reply:
fixed